yaml/water.yaml

```diff
@@ -139,39 +160,39 @@ filters:
       kind: ocean
     table: ne
   - filter: {featurecla: Alkaline Lake}
-    min_zoom: 0
+    min_zoom: *water_polygon_min_zoom
```

Comment:
For Natural Earth features we should just trust their `min_zoom` data property.

Here and elsewhere...

Reply:
Yup, good point. Does this YAML pre-date NE v4? I think we might have been relying on the filter which removes features smaller than a pixel a little too much. Fixed in 67aa978edf2298a6028cdbab8ab762aaf2f59f6a.